perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -913,4 +926,37 @@ sub get_time_spent {
     return $admin_logs ? $admin_logs->get_column('sum_time_spent') : 0;
 }
 
+=head2 get_cobrand_logged
+
+Get a cobrand object for the cobrand the problem was logged for.
+
+e.g. if a problem was logged at www.fixmystreet.com, this will be a
+FixMyStreet::Cobrand::FixMyStreet object.
+
+=cut
+
+sub get_cobrand_logged {
+    my $self = shift;
+    my $cobrand_class = FixMyStreet::Cobrand->get_class_for_moniker( $self->cobrand );
+    return $cobrand_class->new;
+}
+
+=head2 get_cobrand_body_handler
+
+Get a cobrand object for the body the problem has been logged against.
+
+e.g.
+    * if the problem was logged at fix.bromley.gov.uk it will return ::Bromley
+    * if a problem was logged at www.fixmystreet.com for the Bromley area, it
+      will *also* return ::Bromley
+
+=cut
+
+sub get_cobrand_body_handler {
+    my $self = shift;
+    my $cobrand_class = FixMyStreet::Cobrand->get_class_for_moniker( $self->cobrand );
```

Comment:
Does this have to be via the moniker, or can we just use whatever we are on (as the corresponding code is in UK.pm)? I think this can be simplified, I mean.